app/src/main/java/com/arygm/quickfix/ui/navigation/BottomNavigationMenu.kt

```diff
@@ -109,7 +109,8 @@ fun BottomNavigationMenu(
 
   // LaunchedEffect allowing to update the bottom bar accordingly to navigationActions
   LaunchedEffect(currentRoute) {
-    val selectedItemId = getBottomBarId(currentRoute) // Get the ID of the selected item
+    val selectedItemId =
+        getBottomBarId(navigationActions.currentRoute()) // Get the ID of the selected item
```

Comment:
Great catch 👍🏼 (I had some problems with the currentRoute on some implementations of screens and I hope this change will fix the problem)